### ISSUE TITLE
Spigot 1.8.3 issues

### ISSUE THREAD

**Issue by Xericore** [Boosik/boosCooldowns#8]
Hi, I just built the plugin and tried it on spigot-1.8.3. Unfortunately it doesn't work properly.
These are the commands I added to the config:
```
      /f home:
        warmup: 10 seconds
        cooldown: 120 seconds
      /spawn:
        warmup: 10 seconds
        cooldown: 120 seconds
```
When I execute /spawn or /f home, it fires immediately. When I try it again, the cooldown starts correctly. And then if I execute it again, it works without the cooldown or warmup. So every 2nd time the warmup/cooldown doesn't work.
There are no errors or messages in the server console or player chat regarding this issue.

Another problem:
I type e.g. /money or any other command like /list and the command works normally. But when I type /spawn or /f home after that, I see "Execution of command /money was cancelled." in the player chat window.

Is this source code the base for the premium plugin here? http://www.spigotmc.org/resources/booscooldowns.2759/

Are you LordBoos on spigotmc? I wanted to test it out before buying it at spigotmc but didn't write anything in the discussion board to avoid hurting your sales there by informing people about this page.
Does the version on spigotmc work correctly on 1.8.3?

**Comment by Boosik** [comment]
Hello, 

I'm LordBoos on spigotmc and this actually is http://www.spigotmc.org/resources/booscooldowns.2759/ source code. Of of bugs you mentioned, they are already fixed in latest version (3.9.8c). Source you compiled was a bit outdated (3.9.8) and contained bugs. Try to recompile with latest commit I've just pushed and it should work without problems.

My plugin should work regardless of minecraft version (any one with UUID support).

**Comment by Xericore** [comment]
Thank you very much for the quick response. I'll give it a try today in the evening.
May I suggest that you adapt the price to 2.99? If it works in the grocery store, it could work for you as well and help you boost your sales. :)
